target/linux/ath79/image/generic.mk

```diff
@@ -56,6 +56,7 @@ endef
 define Build/engenius-tar-gz
 	-[ -f "$@" ] && \
 	mkdir -p $@.tmp && \
+	touch $@.tmp/failsafe.bin && \
```

Comment:
Since this affects several devices, it should go into a separate commit with proper description.